code/modules/vehicles/mecha/combat/durand.dm

```diff
@@ -189,12 +189,12 @@ own integrity back to max. Shield is automatically dropped if we run out of powe
 	if(switching && !signal_args[1])
 		return
 	if(!chassis.defense_mode && (!chassis.cell || chassis.cell.charge < 100)) //If it's off, and we have less than 100 units of power
-		to_chat(currentuser, "[icon2html(src, currentuser)]<span class='warn'>Insufficient power; cannot activate defense mode.</span>")
+		chassis.balloon_alert(owner, "Insufficient power; cannot activate defense mode.")
```

Comment:
```suggestion
		chassis.balloon_alert(owner, "insufficient power")
```

1. Arm has made them lowercase, and I've decided I like that.
2. We already know we're activating the shield, so it's unnecessary to state it again. Normally it wouldn't matter, but balloon alrets are best kept short.